draft-ietf-tls-tls13.md

```diff
@@ -2159,7 +2176,15 @@ Although X9.62 supports multiple point formats, any given curve
 MUST specify only a single point format. All curves currently
 specified in this document MUST only be used with the uncompressed
 point format (the format for all ECDH functions is considered
-uncompressed).
+uncompressed). Peers MUST validate each other's public value Y by ensuring
+that the point is a valid point on the elliptic
+curve. For all elliptic curves except for x25519 and x448, the appropriate
```

Comment:
I checked the X25519 specification, and it does include an *optional* check in Section 6.1 that detects when the DH share is in a small subgroup. My proposal is to cite the X25519/X448, and specify that the small subgroup check should be mandatory. 

The argument is that even if TLS does not specifically require this behavior, many different applications *around* TLS (e.g., channel bindings, RFC 5929) may assume contributory behavior. This is an inexpensive check that may kill a whole class of attacks.